lib/recurly-data.js

```diff
@@ -7,6 +7,7 @@ const _ = require('lodash')
 const rparser = require('./parser.js')
 const debugR = require('debug')('recurring:request')
 const debug = require('debug')('recurring')
+const pkg = require('../package.json')
```

Comment:
@bhelx I know there is https://github.com/indexzero/node-pkginfo which is designed specifically for this sort of thing. I question wether the overhead of an extra dependency is worth it though. The way you have done it is pretty standard.